### PR TITLE
Update all the deps to the latest versions

### PR DESCRIPTION
**Pull request by vedang**
(and a minor fix in the inspect alias comments as well)

The commit message contains detailed information on how the deps were updated

### CODE REVIEW

**Comment by practicalli-john** [comment]
Thank you for offering this pull request.

Unfortunately there are some aliases that shouldn't be updated, notably those including tools.deps.alpha which uses the addlibs3 branch.

Changes are defined in the changeling, rather than a commit message, for greater visibility

**Comment by vedang** [comment]
Just saw your reply now. I'll make the changes you've mentioned and updated the commit for review tomorrow!

**Comment by vedang** [comment]
I've made the requested changes, please re-review! Thanks!

File: deps.edn [hunk 1]
```diff
@@ -181,11 +181,11 @@
   ;; Requires nashorn in a version of ClojureScript
   ;; WARNING: stack trace - does not run TO DEPRECATE
   :repl/rebel-figwheel
-  {:extra-deps {org.clojure/clojurescript       {:mvn/version "1.9.946"} ;; version contains cljs.repl.nashorn
+  {:extra-deps {org.clojure/clojurescript       {:mvn/version "1.11.54"} ;; version contains cljs.repl.nashorn
```
Comment (practicalli-john) [inline deps.edn:184]: This alias is set to a version of Clojurescript that contains cljs.repl.nashorn, which is needed for this alias to work correctly.  Updating the version breaks the alias (and the alias is marked as deprecated, so should not be changed)

File: deps.edn [hunk 2]
```diff
@@ -108,34 +108,34 @@
...
+                cider/cider-nrepl             {:mvn/version "0.28.4"}
                 refactor-nrepl/refactor-nrepl {:mvn/version "3.5.2"}}
    :main-opts  ["-m" "nrepl.cmdline"
                 "--middleware" "[refactor-nrepl.middleware/wrap-refactor,cider.nrepl/cider-middleware]"
                 "-i"]}
 
   ;; clojure -M:repl/cider-cljs
   :repl/cider-cljs
-  {:extra-deps {org.clojure/clojurescript       {:mvn/version "1.10.773"}
+  {:extra-deps {org.clojure/clojurescript       {:mvn/version "1.11.54"}
```
Comment (practicalli-john) [inline deps.edn:126]: TODO: test version of ClojureScript works okay.  Some recent versions of ClojureScript were causing issues and kept back to 1.10.773

File: deps.edn [hunk 3]
```diff
@@ -108,34 +108,34 @@
...
+                cider/cider-nrepl               {:mvn/version "0.28.4"}
                 cider/piggieback                {:mvn/version "0.5.3"}}
    :main-opts  ["-m" "nrepl.cmdline"
                 "-i"
                 "--middleware" "[cider.nrepl/cider-middleware,cider.piggieback/wrap-cljs-repl]"]}
 
   :repl/figwheel
-  {:extra-deps {org.clojure/clojurescript       {:mvn/version "1.10.773"}
-                com.bhauman/figwheel-main       {:mvn/version "0.2.16"}
+  {:extra-deps {org.clojure/clojurescript       {:mvn/version "1.11.54"}
```
Comment (practicalli-john) [inline deps.edn:135]: TODO: This version of ClojureScript requires testing with figwheel-main to ensure it works

File: deps.edn [hunk 4]
```diff
@@ -551,16 +551,16 @@
...
 
   :inspect/portal-cli
-  {:extra-deps {djblue/portal {:mvn/version "0.23.0"}}}
+  {:extra-deps {djblue/portal {:mvn/version "0.26.0"}}}
 
   :inspect/portal-web
-  {:extra-deps {djblue/portal             {:mvn/version "0.23.0"}
-                org.clojure/clojurescript {:mvn/version "1.10.844"}}
+  {:extra-deps {djblue/portal             {:mvn/version "0.26.0"}
+                org.clojure/clojurescript {:mvn/version "1.11.54"}}
```
Comment (practicalli-john) [inline deps.edn:558]: TODO: ClojureScript version will need to be tested with Portal to ensure there are no issues.  Portal is still showing ClojureScript 1.10.844, although the README may simple have not been updated.

File: CHANGELOG.org [hunk 5]
```diff
@@ -2,6 +2,27 @@
 
 * Unreleased
 
+* 2022-05-25
+** Changed
+*** Update library versions using `clojure -T:search/outdated > outdated_$(date -u +%Y-%m-%d).org` command
+
+  | :name                                |   :current |    :latest |
+  |--------------------------------------+------------+------------|
+  | clj-kondo/clj-kondo                  | 2022.04.08 | 2022.04.25 |
```
Comment (practicalli-john) [inline CHANGELOG.org:11]: The updated version of clj-kondo in the GitHub action workflow does not seem to be included in this PR.  

TODO: Investigate if antq is still detecting for new versions in GitHub actions.

**Comment by practicalli-john** [comment]
I've made some notes, as much for myself, especially around checking newer versions of ClojureScript.  I've held most aliases at a known working version of ClojureScript as there were issues in recent releases.  These issues may be resolved, but some testing of the respective aliases is definitely required.

File: CHANGELOG.org (hunk 5 above)
Comment (practicalli-john) [reply CHANGELOG.org:11]: antq is showing there is a new version of clj-kondo in the GitHub workflow, so this is updated along with other libraries to keep things consistent.

**Comment by practicalli-john** [comment]
I've pushed an update without the ClojureScript version changes for now, as I dont have much time at the moment to properly test them (and they work as they are)

**Comment by vedang** [comment]
Thank you! Sorry to push more work to your plate, I think it'll be useful to note these conflicts as comments in the actual file, so that the next time someone else updates the deps they know which parts to ignore!

**Comment by vedang** [comment]
Should I close this PR? Or is there anything you'd like me to add / change in these commits?

**Comment by practicalli-john** [comment]
I'll create a pull request template that will inform people not to do sweeping changes to this repository, especially updates to libraries without a way to demonstrate those changes have been tested.

Thank you.
